src/OpenTelemetry/Trace/Export/ActivityProcessor.cs

```diff
@@ -42,5 +42,12 @@ public abstract class ActivityProcessor
         /// <param name="cancellationToken">Cancellation token.</param>
         /// <returns>Returns <see cref="Task"/>.</returns>
         public abstract Task ShutdownAsync(CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Flushes all activity objects that have been queued for processing.
```

Comment:
minor: `Export all ended spans to the configured Exporter that have not yet been exported.` - This is the spec description. Lets use something on that line. "queue" is not necessarily present for all processor.